VisualizationBase/src/overlays/ZoomLabelOverlay.cpp

```diff
@@ -132,6 +135,11 @@ const TextStyle* ZoomLabelOverlay::associatedItemTextStyle() const
 
 QList<Item*> ZoomLabelOverlay::itemsThatShouldHaveZoomLabel(Scene* scene)
 {
+	// TODO at the moment only one viewItem is used for diffs, maybe changes in future
+	// disable zoom labels for diff views
+	if (Visualization::VisualizationManager::instance().mainScene()->currentViewItem()->name().contains("DiffView"))
```

Comment:
Hmm, I'm not sure I like this change. We need a more principled approach, not just checking the name (which could also be user provided). If you want this, add a bool field to `ViewItem` that enables/disables zoom labels and set that field to true for the diff case, and here, just check the field.